packages/client/wallets/aa/src/blockchain/wallets/FireblocksNCWallet.ts

```diff
@@ -174,3 +185,30 @@ const getDefaultAlgorithems = (): Set<TMPCAlgorithm> => {
     algorithms.add("MPC_CMP_ECDSA_SECP256K1");
     return algorithms;
 };
+
+function storeNCWDataToLocalStorage(walletId: string, deviceId: string) {
+    localStorage.setItem(NCW_DEVICE_ID_KEY, deviceId);
+    localStorage.setItem(NCW_WALLET_ID_KEY, walletId);
+}
+
+function getNCWDataFromLocalStorage(): { walletId: string; deviceId: string } | null {
+    const deviceId = localStorage.getItem(NCW_DEVICE_ID_KEY);
+    const walletId = localStorage.getItem(NCW_WALLET_ID_KEY);
+    if (deviceId && walletId) {
+        return {
+            walletId,
+            deviceId,
+        };
+    } else {
+        return null;
+    }
+}
+
+function storeAddressToLocalStorage(address: string) {
+    localStorage.setItem(NCW_ADDRESS_KEY, address);
+}
+
+function getAddressFromLocalStorage(): string | null {
+    const address = localStorage.getItem(NCW_ADDRESS_KEY);
+    return address ?? null;
```

Comment:
isn't `return address` enough?
Actually we can delete this function and call `localStorage.getItem(NCW_ADDRESS_KEY);` when needed

Reply:
Totally agree. Change pushed.